cmd/zpool/zpool_iter.c

```diff
@@ -128,8 +131,9 @@ pool_list_get(int argc, char **argv, zprop_list_t **proplist, int *err)
 
 	zlp->zl_proplist = proplist;
 
+	void *data[] = {zlp, (void *)literal};
```

Comment:
Why don't we instead add a `zl_literal` boolean_t to the `zpool_list` structure in the same way one was added for the `zl_proplist`.

Reply:
Good point! Updated.